src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,18 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
-);
+export const MovieList = ({ movies }) => {
+  const moviesExist = movies.length > 0;
+
+  return (
+    <div className="movies">
+      {moviesExist ? (
+        movies.map(movie => (
+          <MovieCard movie={movie} key={movie.imdbId} />
+        ))
+      ) : (
+        <div>No movies available</div>
+      )}
+    </div>
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings
```suggestion
{moviesExist ? (
    <ul className="movies">
        {movies.map(movie => (
          <MovieCard movie={movie} key={movie.imdbId} />
        ))}
    </ul>
     ) : (
        <div>No movies available</div>
     )}
```